samples/Playground/Directory.Packages.props

```diff
@@ -23,12 +23,12 @@
 		<PackageVersion Include="Uno.Core.Extensions.Logging.Singleton" Version="4.0.1" />
 		<PackageVersion Include="Uno.Extensions.Logging.OSLog " Version="1.4.0" />
 		<PackageVersion Include="Uno.Extensions.Logging.WebAssembly.Console" Version="1.3.0" />
-		<PackageVersion Include="Uno.Material" Version="2.3.0" />
-		<PackageVersion Include="Uno.Material.WinUI" Version="2.3.0" />
-		<PackageVersion Include="Uno.Toolkit.UI" Version="2.4.0-dev.114" />
-		<PackageVersion Include="Uno.Toolkit.UI.Material" Version="2.4.0-dev.114" />
-		<PackageVersion Include="Uno.Toolkit.WinUI" Version="2.4.0-dev.114" />
-		<PackageVersion Include="Uno.Toolkit.WinUI.Material" Version="2.4.0-dev.114" />
+		<PackageVersion Include="Uno.Material" Version="2.4.1" />
+		<PackageVersion Include="Uno.Material.WinUI" Version="2.4.1" />
```

Comment:
We might have issue building the WinAppSdk apps that are referencing the latest stable Uno.Material packages. There should be a backport coming soon but if you end up seeing things like:

> error : PRI175: 0x80073b0f - Processing Resources failed with error: Duplicate Entry.
> error : PRI277: 0x80073b0f - Conflicting values for resource 'Files/Uno.Material/Styles/Application/v2/SharedColors.xbf'

it's because of that